src/components/MessagesList/MessagesList.vue

```diff
@@ -273,8 +288,11 @@ export default {
 		messagesList: {
 			immediate: true,
 			handler(messages) {
```

Comment:
```suggestion
			handler(newMessages, oldMessages) {
```
For easier check for token / empty array

Reply:
aa so to check tokens of new and old ? but at least in softupdate, you only parse one list and the other one is computed already :p

Reply:
You don't even need to parse them, could just check if oldMessages[0] exists and `newMessages[0]?.token === oldMessages[0]?.token`